core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -11,8 +11,7 @@ trait SubmissionCostService {
 
   val stringParamType: QueryParameterType
 
-  def getSubmissionCosts(submissionId: String,
```

Comment:
Was `submissionId` just never used?

Reply:
removed `executeSubmissionCostsQuery ` in [c82db18](https://github.com/broadinstitute/rawls/pull/3474/commits/c82db189a5b9f86f447fb8413b8f94c574d87ac9)